template/.github/workflows/ci.yaml

```diff
@@ -1,41 +1,38 @@
 name: CI
 
 on:
-  - push
-  - pull_request
+  pull_request:
+  push:
+    branches: [ main ]
```

Comment:
How is it related to the PR?…

Reply:
Leaving it as it was, any gem generated from the template would run tests twice on PR, which didn't feel right.
It was an easy fix while I was changing the file 😄